junit-runtime/src/main/scala/com/novocode/junit/OutputCapture.scala

```diff
@@ -0,0 +1,47 @@
+package com.novocode.junit
+
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+object OutputCapture {
+  def start(): OutputCapture = new OutputCapture()
+
+  private def getScalaOut(): PrintStream = {
+    try {
+      scala.Console.out
+    } catch {
+      case t: Throwable => null
```

Comment:
Can this be a bit more precise than `Throwable`? What is this supposed to protect against?